app/javascript/gobierto_data/webapp/components/sets/data/SQLEditorCode.vue

```diff
@@ -107,32 +110,53 @@ export default {
     // metaKey + keyUp doesn't work with MacOS
     // https://stackoverflow.com/questions/11818637/why-does-javascript-drop-keyup-events-when-the-metakey-is-pressed-on-mac-browser
     this.editor.on("keydown", this.onKeyDown);
+    // This event detects any changes in the editor
+    this.editor.on("change", this.onChange);
+    // We need this event to enable autocomplete(hint)
+    this.editor.on("keyup", this.onKeyUp);
   },
   methods: {
+    onKeyUp(editor, e) {
+      const {
+        state: {
+          completionActive: isEditorActive
+        }
+      } = editor
+
+      // Show autocomplete only when a letter key is pressed
+      if (!isEditorActive && e.keyCode > 64 && e.keyCode < 91) {
+        editor.showHint({ completeSingle: false })
+      }
+
+      this.$root.$emit('enableSavedButton')
+    },
+    onChange(editor) {
+      this.mergeTables();
+      const value = editor.getValue();
+      this.$root.$emit("setCurrentQuery", value);
```

Comment:
You're updating the query only `onchange`event. Is that what we want? I say so, because as far as I know, `onchange` is triggered only when you *blur* the editor, so ctrl+enter event does not run the query properly

Reply:
`change` event for CodeMirror is different to `onchange`

> "change" (doc: CodeMirror.Doc, changeObj: object)
    Fired whenever a change occurs to the document. changeObj has a similar type as the object passed to the editor's "change" event.

[documentation](https://codemirror.net/doc/manual.html#event_doc_change)